microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -23,63 +23,55 @@ ms.date: 02/24/2023
+There are several different levels of testing that you can do to test your site under different conditions:
 
-*  Device emulators and simulators enable you to mimic your development site on a range of devices, from your workstation.
+* **Browser emulators** help you to quickly test how your website looks and behave at different screen sizes and resolutions, with touch event emulation, and by overriding the user-agent string the browser sends. You use browser emulators from the browser you're developing on. See [Browser emulators](#browser-emulators), below.
 
-*  Cloud-based emulators enable you to automate unit tests for your site across different platforms.
+* **Device emulators and simulators** enable you to run other devices from the device you're developing on and test your website on these other devices. See [Device emulators and simulators](#device-emulators-and-simulators), below.
+
+* **Cloud-based emulators** enable you to run automated tests for your site across different platforms. See [Cloud-based emulators and simulators](#cloud-based-emulators-and-simulators), below.
 
 
 <!-- ====================================================================== -->
 ## Browser emulators
 
-Browser emulators are great for testing the responsiveness of a site.  But a browser emulator doesn't emulate differences in API, CSS support, and certain behaviors that manifest only on a mobile browser on an actual device.  Test your site on browsers running on real devices, to be certain everything behaves as expected.
-
-
-<!-- ------------------------------ -->
-#### Firefox Responsive Design View
-
-Firefox has a [responsive design view](https://developer.mozilla.org/docs/Tools/Responsive_Design_View) that encourages you to stop thinking in terms of specific devices and instead explore how your design changes at common screen sizes, or on your own screen size by dragging the edges of the window.
+Browser emulators are great for testing the way your site looks and behaves at different screen sizes and resolutions, or when touch events are enabled, or when the user-agent string is set to another browser's user-agent string.
 
+You use browser emulators from the browser you're developing on. For example, in Microsoft Edge, you can use the **Device Emulation** tool in DevTools to get an approximation of how your site looks and behaves on different devices.  See [Emulate mobile devices (Device Emulation)](./index.md) for more information on how to use the **Device Emulation** tool in Microsoft Edge DevTools.
 
-<!-- ------------------------------ -->
-#### EdgeHTML emulation
+Browser emulators don't emulate differences in web API or CSS support. For example, the **Device Emulation** tool in Microsoft Edge doesn't emulate how the different web APIs or CSS features that your site might be using are supported in Firefox or Safari. Test your site on browsers running on real devices, to be certain everything behaves as expected.
```

Comment:
```suggestion
Browser emulators don't emulate differences in web API or CSS support. For example, the **Device Emulation** tool in Microsoft Edge doesn't emulate how the different web APIs or CSS features that your site might be using are supported in Firefox or Safari.  To be certain everything behaves as expected, test your site on browsers running on real devices.
```
imo it's a little better here to use pattern: To X, do Y.  instead of: Do X, to Y.  tho either way works here imo.
@captainbrosset

Reply:
applied locally, will commit/push